src/SearchField/README.md

```diff
@@ -77,6 +77,15 @@ notes:
 />
 ```
 
+### With the submit button outside the input
+
+```jsx live
+<SearchField
+  submitButtonLocation="external"
+  onSubmit={value => console.log(`search submitted: ${value}`)}
+/>
+```
+
 ### Advanced Usage
```

Comment:
Is there an example of using `SearchField.Advanced with an external button location? That might be a good one to include here.

Related, is the `disabled` prop on `SearchField` and/or `SearchField.Advanced` working? As far as I can tell, adding `disabled` isn't actually disabling the form / button.

Reply:
Yes, right, I add `submitButtonLocation` example with the `SearchField.Advanced` and I add disabled working on the `SeachField.Advanced`.